spec/acceptance/foreman_cli_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper_acceptance'
+
+describe 'Scenario: install foreman-cli without foreman' do
+  before(:context) do
+    case fact('osfamily')
+    when 'RedHat'
+      on default, 'yum -y remove foreman* && rm -rf /etc/yum.repos.d/foreman*.repo'
```

Comment:
Are there other packages, such as `tfm-*` (or more specifically, `tfm-rubygem-hammer_cli_*`) that should be removed too, to ensure a consistent state?